File: MiniProyecto/ec/edu/utpl/poo/MiniProyetco/RunTienda.py
from Tienda import Tienda
if __name__ == '__main__':


    T1=Tienda("Moto",-6,100)
    T1.imprimirLista()

    T1.agregarProducto("Carro",3,2000)
    T1.imprimirLista()

    T2 = Tienda()
    T2.imprimirLista()

    T3=Tienda("Motoneta",2,100)
    T3.imprimirLista()

File: java/src/ec/edu/utpl/poo/model/RunTienda.java
package ec.edu.utpl.poo.model;

public class RunTienda {
    public static void main(String[] args) {


        Tienda T1 = new Tienda();
        System.out.println(T1.getCantidad());
        T1.imprimir();
        T1.agregarProductos("Moto",3,200);
        T1.imprimir();

        Tienda T2 = new Tienda("Carro",5,300);
        T1.imprimir();

        Tienda T3= new Tienda("Patineta",12,100);
        T3.imprimir();
    }
}
